### DISCUSSION TITLE
V5下还有 load_balance 轮询功能吗？

### DISCUSSION THREAD

**Discussion by deddey** [IrineSistiana/mosdns#583] [General]
我有电信和联通两条出口，使用不同ecs 去请求，通过 load_balance 能基本达到负载的效果。
现在V5是不是没有 load_balance 了，还是说在V5中换成了其他的使用方法实现？

**Comment by urlesistiana** [comment]
https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns-v5/ru-he-pei-zhi-mosdns/sequence-cha-jian#random

**Comment by deddey** [comment]
谢谢你的回复。
大概明白了，通过 random 实现按指定概率选择后面的操作。
如果需要在三个或以上的选项中轮询，这个感觉稍微复杂点，但能实现了负载比例的调整，要赞一个。
